File: 25372.cpp
#include <iostream>

using namespace std;

int main()
{
    int n = 0, b = 0;
    string a;
    cin >> n;
    for(int i = 1; i <= n; i++)
    {
        cin >> a;
        b = a.length();
        if(b >= 6 && b <= 9)
        {
            cout << "yes" << endl;
        }
        else cout << "no" << endl;
    }
    return 0;
}

File: 25372.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        String password;
        Scanner scanner = new Scanner(System.in);
        int intValue = scanner.nextInt();;

        for(int i = 1; i <= intValue; i++)
        {
            password = scanner.next();

            if(password.length() >= 6 && password.length() <= 9)
            {
                System.out.println("yes");
            }
            else System.out.println("no");
        }
    }
}
